bumping-ios-build-number.md

```diff
@@ -0,0 +1,27 @@
+# Bumping iOS Build Number
+
+#### 🤔 What Happened
+
+If you're submitting your iOS app to App Store using the `eas submit` command, you may encounter the error saying you've already submitted a particular version of the app.
+
+#### 💡 Solution
+
+1. - **If you are building your app using EAS Build**
+
+     Increment the value for the `expo.ios.buildNumber` key in `app.json`. Remember that this value must be a string. Optionally, commit this change. Then, build a new app archive with `eas build`. Wait for the build to finish.
```

Comment:
this won't actually update the build number for a bare react-native project that runs on eas build, at the moment. it'll only work for *managed* projects.